src/PresentationalComponents/Charts/SummaryChart/SummaryChartItem.js

```diff
@@ -1,45 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import './SummaryChartItem.scss';
+import { Link } from 'react-router-dom';
+import { Battery } from '@red-hat-insights/insights-frontend-components';
+import { Split, SplitItem, StackItem } from '@patternfly/react-core';
 
 const SummaryChartItem = (props) => {
-    const numIssues = props.numIssues;
-    let percentage = ((numIssues / props.totalIssues) * 100).toFixed(1);
-    let lowerCaseName = props.name.toLowerCase();
-
-    let barWidth = { width: percentage + '%' };
+    const { numIssues, name, riskName } = props;
+    const returnLink = (children) => <Link to={ `/actions/${riskName}` }> { children } </Link>;
 
     return (
-        <li widget-type='InsightsSummaryChartItem' widget-id={ props.name }>
-            { numIssues > 0 &&
-            <React.Fragment>
-                <div className='metrics'>
-                    <strong>{ props.name } </strong>
-                    <span className='num'>({ percentage }%)</span>
-                </div>
-                <div className='progress-bars'>
-                    <div
-                        className= { 'progress-bar progress-bar-' + lowerCaseName }
-                        style= { barWidth }
-                    ></div>
-                    <div className='bar'></div>
-                </div>
-            </React.Fragment>
-            }
-            { numIssues === 0 &&
-            <div className='no-errors'>
-                <i className='fas fa-check-circle small-spacer green'></i>
-                You have no issues of { lowerCaseName } severity
-            </div>
-            }
-        </li>
+        <StackItem widget-type='InsightsSummaryChartItem' widget-id={ name }>
+            <Split>
+                <SplitItem className='pf-u-pl-md pf-u-pr-md'>
+                    { returnLink(<Battery aria-label={ riskName } label={ riskName } severity={ name.toLowerCase() } labelHidden={ true }/>) }
+                </SplitItem>
+                <SplitItem className='pf-u-text-align-right pf-u-pl-sm'>
+                    { returnLink(numIssues) }
+                </SplitItem>
+                <SplitItem className='pf-u-text-align-left pf-u-pl-sm'>
+                    { returnLink(`${name} affecting ${props.affectedSystems} systems`) }
```

Comment:
The `numIssues` needs to be inside of the same item as the `affecting foo systems`. You also shouldn't be returning two separate links. The whole `1 High affecting 9 systems` should be 1 link instead of 2.